<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q::vtk::InteractorStyleSeedsPlacer</class>
 <widget class="QWidget" name="Q::vtk::InteractorStyleSeedsPlacer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SCBInteractorStyleSeedsPlacer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxX"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Y: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxY"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxZ"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetSeeds"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxNavigation">
     <property name="text">
      <string>Navigation</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonAddOne">
       <property name="text">
        <string>Add One</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteOne">
       <property name="text">
        <string>Delete One</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteAll">
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
